### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Oleg-Chashko**
- Added HTML Element line break **< br >**
- Removed empty lines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,7 @@ Detection Script to help identify why your PC isn't Windows 11 Release Ready.
 ----
 
 ## Download
-
-[Download latest stable release](https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe)
-
+[Download latest stable release](https://github.com/rcmaehl/WhyNotWin11/releases/latest/download/WhyNotWin11.exe)<br>
 [Download latest testing release](https://nightly.link/rcmaehl/WhyNotWin11/workflows/wnw11/main/WNW11.zip)\
```
Comment (micwoj92) [inline README.md:18]: Why not just use backslash as here?

**Review by micwoj92 — CHANGES_REQUESTED** [review]

**Comment by Oleg-Chashko** [comment]
What difference does it make, the result is the same. In Markdown I use < br > or < br/ >. The main thing is the result? If you don't like it, I can close the request.

**Comment by Oleg-Chashko** [comment]
So why didn't you use this when you were writing the Markdown code?

**Comment by Oleg-Chashko** [comment]
https://www.markdownguide.org/basic-syntax/#line-break-best-practices

Screen Shot 2021-07-20 at 23 06 09

**Comment by micwoj92** [comment]
> So why didn't you use this when you were writing the Markdown code?

I just didn't think of it.

> What difference does it make, the result is the same. In Markdown I use < br > or < br/ >. The main thing is the result? If you don't like it, I can close the request.

None, just consistency.

> Screen Shot 2021-07-20 at 23 06 09

GitHub uses Commonmark and WNW11 is hosted only on GitHub, so I don't see that as a problem.

**Comment by Oleg-Chashko** [comment]
I do not see this as a problem either. :))

**Comment by micwoj92** [comment]
Oh but the whitespace changes and rest were good. If you just replaced <br> with / then it was good to merge

**Pull request closed without merging**